okio/src/commonMain/kotlin/okio/internal/-Buffer.kt

```diff
@@ -1297,7 +1296,7 @@ internal inline fun Buffer.commonIndexOf(bytes: ByteString, fromIndex: Long): Lo
     while (offset < resultLimit) {
       // Scan through the current segment.
       val data = s.data
-      val segmentLimit = okio.minOf(s.limit, s.pos + resultLimit - offset).toInt()
```

Comment:
This one was interesting; we had our own minOf for mixed parameter types. Is this no longer necessary?

Reply:
Just simplify the package name, it has been imported.